x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/data/ConstantNullBlock.java

```diff
@@ -98,7 +100,7 @@ public Block expand() {
 
     @Override
     public long ramBytesUsed() {
-        return BASE_RAM_BYTES_USED;
+        return 0L;
```

Comment:
Doesn't it still use the memory?

Reply:
D'oh! yeah, it's marginal. I restored it.